libraries/matchers.rb

```diff
@@ -14,4 +14,12 @@ def enable_gluster_mount(resource_name)
   def disable_gluster_mount(resource_name)
     ChefSpec::Matchers::ResourceMatcher.new(:gluster_mount, :disable, resource_name)
   end
+
+  def set_gluster_volume_option(resource_name) # rubocop:disable Style/AccessorMethodName
```

Comment:
Can you actually put the rubocop rule in `.rubocop.yml` as an excludes for this file?